src/Avalonia.Animation/Animation.cs

```diff
@@ -7,68 +7,209 @@
-        public PlaybackDirection PlaybackDirection { get; set; }
+        public PlaybackDirection PlaybackDirection
+        {
+            get { return _playbackDirection; }
+            set { SetAndRaise(PlaybackDirectionProperty, ref _playbackDirection, value); }
+        }
 
         /// <summary>
         /// Gets or sets the value fill mode for this animation.
-        /// </summary>
-        public FillMode FillMode { get; set; }
+        /// </summary> 
+        public FillMode FillMode
+        {
+            get { return _fillMode; }
+            set { SetAndRaise(FillModeProperty, ref _fillMode, value); }
+        }
 
         /// <summary>
         /// Gets or sets the easing function to be used for this animation.
         /// </summary>
-        public Easing Easing { get; set; } = new LinearEasing();
-
-        /// <summary>
-        /// Gets or sets the speed multiple for this animation.
-        /// </summary>
-        public double SpeedRatio { get; set; } = 1d;
+        public Easing Easing
+        {
+            get { return _easing; }
+            set { SetAndRaise(EasingProperty, ref _easing, value); }
+        }
 
         /// <summary> 
-        /// Gets or sets the delay time for this animation. 
+        /// Gets or sets the initial delay time for this animation. 
         /// </summary> 
-        /// <remarks>
-        /// Describes a delay to be added before the animation starts, and optionally between 
-        /// repeats of the animation if <see cref="DelayBetweenIterations"/> is set. 
-        /// </remarks> 
-        public TimeSpan Delay { get; set; }
+        public TimeSpan Delay
+        {
+            get { return _delay; }
+            set { SetAndRaise(DelayProperty, ref _delay, value); }
+        }
 
         /// <summary> 
-        /// Gets or sets a value indicating whether <see cref="Delay"/> will be applied between 
-        /// iterations of the animation.
+        /// Gets or sets the delay time in between iterations.
+        /// </summary> 
+        public TimeSpan DelayBetweenIterations
+        {
+            get { return _delayBetweenIterations; }
+            set { SetAndRaise(DelayBetweenIterationsProperty, ref _delayBetweenIterations, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the speed multiple for this animation.
         /// </summary> 
-        /// <remarks>
-        /// If this property is not set, then <see cref="Delay"/> will only be applied to the first 
-        /// iteration of the animation. 
-        /// </remarks> 
-        public bool DelayBetweenIterations { get; set; }
+        public double SpeedRatio
+        {
+            get { return _speedRatio; }
+            set { SetAndRaise(SpeedRatioProperty, ref _speedRatio, value); }
+        }
+
+        /// <summary>
+        /// Obselete: Do not use this property, use <see cref="IterationCount"/> instead.
+        /// </summary>
+        /// <value></value>
+        [Obsolete]
```

Comment:
Can you talk about why it is obsolete and how to move to other property in `[Obsolete]`?